conboarding.md

```diff
@@ -116,8 +116,48 @@ How much does it cost to run things?
       and [John Hudson's course notes](https://rcweb.dartmouth.edu/~john/HPC/).
    4. Request `rc-DBIC` group from Yarik. (Necessary to use Datalad) Be sure to follow this guide to use DBIC-installed git-annex. https://dbic-handbook.readthedocs.io/en/latest/mri/dataaccess.html#discovery-filesystem
 
-3. ReproNim: request iam from David for AWS Access
-4. DANDI: request credentials for DANDI from Satra
+3. VNC is available on `typhon`
```

Comment:
just replace `typhon` with some `SERVERNAME` ;-)  no need to disclose such things too specifically and instructions should generalize for any SERVERNAME really .

Reply:
done